extensions/ql-vscode/test/unit-tests/data-extensions-editor/auto-model-v2.test.ts

```diff
@@ -80,3 +83,106 @@ describe("createAutoModelV2Request", () => {
     expect(parsed).toEqual(result.candidates);
   });
 });
+
+describe("getCandidates", () => {
+  it("doesnt return methods that are already modelled", () => {
+    const externalApiUsages: ExternalApiUsage[] = [];
```

Comment:
Any reason you're initialising and then doing `push`? Can just do it in one line.

```ts
const externalApiUsages: ExternalApiUsage[] = [ 
  {
    ...
  } 
];
```